src/selectors/markdownSelectors.test.js

```diff
@@ -0,0 +1,19 @@
+import { getMarkdown } from './markdownSelectors';
+
+describe('selector tests', () => {
+  it('test the get single markdown selector', () => {
```

Comment:
it test the get single markdown selector is not a good sentence. maybe

it gets markdown text from state.